great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -37,6 +43,115 @@ def __init__(
             data_context=data_context,
         )
 
+    def _plot_descriptive(self, line: alt.Chart):
```

Comment:
@NathanFarmer Can all this go into result object generically?  As in "generic line chart".

Reply:
@alexsherstinsky following our conversation generic charts have been moved into `DataAssistant`.

Reply:
✅ @alexsherstinsky I moved what I could